code/main_0417.py

```diff
@@ -179,24 +157,13 @@
                             H_inv: weight_cov_val})
 
     # compute rff, to compute population-version covariance later
-    rff_output_val = sess.run(rff_output, feed_dict={X: xx,
-                                                     Y_true: test_y,
-                                                     H_inv: weight_cov_val})
-    train_x_val = sess.run(rff_output, feed_dict={X: train_x,
-                                                  Y_true: test_y,
-                                                  H_inv: weight_cov_val})
-    tst_xx_val = sess.run(rff_output, feed_dict={X: test_x,
-                                                 Y_true: test_y,
-                                                 H_inv: weight_cov_val})
-
-
+    rff_output_val = sess.run(rff_output, feed_dict={X: xx})
+    train_x_val = sess.run(rff_output, feed_dict={X: train_x})
+    tst_xx_val = sess.run(rff_output, feed_dict={X: test_x})
 
 ## weight covariance matrix for all data, should equal to weight_tmp
 weight_cov_all = util.compute_inverse(train_x_val, sig_sq=noise_std ** 2)
-np.max(np.abs(weight_tmp - weight_cov_all))
```

Comment:
please verify that this value is very small...after above changes I usually get a value < 1e-6